examples/cognitoidp/lambdatrigger.yaml

```diff
@@ -1,39 +0,0 @@
-apiVersion: lambda.aws.upbound.io/v1beta1
-kind: Function
-metadata:
-  annotations:
-    meta.upbound.io/example-id: s3control/v1beta1/objectlambdaaccesspoint
-  labels:
-    testing.upbound.io/example-name: lambda-example
-  name: lambda-example
-spec:
-  forProvider:
-    packageType: Image
-    image_uri: "123456789012.dkr.ecr.eu-west-2.amazonaws.com/example:0.1"
-    memorySize: 128
-    publish: true
-    region: eu-west-2
-    timeout: 5
-    roleSelector:
-      matchLabels:
-        testing.upbound.io/example-name: lambda-example
-
```

Comment:
an example of using container image in lamda also didn't exist, hence I thought this might be useful example too

Reply:
Thank you for this PR @thekaleidoscope you are right, that would be useful. However, the `lambda.Function` resource is not relevant to this PR.  Also if we want to add an example of this we have to add it under the `lambda` group and the example name should contain the relevant resource, e.g. `function-with-container-image.yaml`.

Feel free to open a new PR adding an example for `lambda.Function` with container image.